metacat-common-server/src/main/java/com/netflix/metacat/common/server/events/MetacatEventBus.java

```diff
@@ -71,7 +76,7 @@ public void postAsync(final ApplicationEvent event) {
      */
     public void postSync(final ApplicationEvent event) {
         log.debug("Received request to post an event {} synchronously", event);
-        CounterWrapper.incrementCounter("metacat.events.sync");
+        registry.counter(LogConstants.CounterEventSync.name()).increment();
```

Comment:
Is it better to do this than to just have a counter reference saved in the constructor as a private member so we don't have to get the name every time? Then we just call increment on the local reference